tests/data.tf

```diff
@@ -0,0 +1,133 @@
+data "aws_partition" "current" {}
+
+data "http" "ip" {
```

Comment:
This is no longer used, might as well remove it.

Reply:
removed